go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/cilium/cilium-cli
 
-go 1.21
+go 1.21.0
 
 // Replace directives from github.com/cilium/cilium. Keep in sync when updating Cilium!
 replace (
```

Comment:
The replace directives need to be updated to what they are in Cilium v1.15.0-pre.0, ie
```
replace (
	go.universe.tf/metallb => github.com/cilium/metallb v0.1.1-0.20220829170633-5d7dfb1129f7

	// Using private fork of controller-tools. See commit msg for more context
	// as to why we are using a private fork.
	sigs.k8s.io/controller-tools => github.com/cilium/controller-tools v0.8.0-1
)
```